test/libsolidity/syntaxTests/nameAndTypeResolution/137_external_visibility.sol

```diff
@@ -3,4 +3,4 @@ contract c {
     function g() public { f(); }
 }
 // ----
-// DeclarationError: (68-69): Undeclared identifier. Did you mean "f"?
+// DeclarationError: (68-69): Undeclared identifier. Did you mean to declare "f" earlier?
```

Comment:
In this case, the error is misleading. Perhaps we should rather change the message to `"f" is not (yet) visible at this point.`